specs/Deployment/DeploymentAPI.md

```diff
@@ -285,15 +459,31 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         DeploymentInitializeOptions();
 
         /// Gets or sets a value that indicates whether the processes associated with the
-        /// WindowsAppSDK main and singleton packages will be shut down forcibly if they are
-        /// currently in use, when registering the WinAppSDK packages.
+        /// WinAppSDK Main and Singleton packages will be shut down forcibly if they are
+        /// currently in use, when initializing the WindowsAppRuntime.
```

Comment:
So when you start my app, you might see some other apps disappear? Is there any kind of interaction with the user first?

Reply:
No. Please see one of the above comments for detailed explanation on ForceDeployment. In short, ForceDeployment only shuts down WinAppSDK packages that are dependent on it's framework packages (ex: Main, Singleton and DDLM). It doesn't shut down any non-WinAppSDK packages that declared dependency on WinAppSDK framework package.